### PR TITLE
Zadání semestrálky pro Showmax

### PR DESCRIPTION
**Pull request by janpipek**
Mohlo by to být takto?

### CODE REVIEW

File: semestralka.adoc [hunk 1]
```diff
@@ -105,3 +105,46 @@ Aplikace bude mít webové rozhraní, přes které uživatel:
 Aplikace musí umožňovat autentizaci proti https://ict.fit.cvut.cz/~web/current/web/ict/GoogleApps/[Google Apps for Education] na FIT. Volitelně by aplikace mohla pracovat se _zdroji_ (https://support.google.com/a/answer/1686462?hl=en[Calendar Resources]), pro alokaci místnosti, ve které se událost koná.
 
 Kontaktní osoba: https://usermap.cvut.cz/profile/vlnasjan/[Jan Vlnas].
+
+
+=== Synchronization between Google and Phabricator calendars
+
+In Showmax, we have two calendar systems which need to be synchronized.
+
+See also https://is.fit.cvut.cz/group/ssp/assignee/assignmentdetail?aid=900[SSP portal and subscribe there] to earn.
```
Comment (hroncok) [inline semestralka.adoc]: to earn money? points? exposure?

Comment (janpipek) [reply semestralka.adoc]: to earn money

**Comment by janpipek** [comment]
Does it look OK now?

**Comment by hroncok** [comment]
Thanks!

**Pull request merged**